selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/PageFactory_Portal/PortalApplicationsStaffing.java

```diff
@@ -29,50 +30,40 @@ public class PortalApplicationsStaffing {
 	private WebElement next;
 
 	public PortalApplicationsStaffing(WebDriver driver) {
-
 		this.driver = driver;
 		PageFactory.initElements(driver, this);
-
+		wait = new WebDriverWait(driver, Duration.ofSeconds(20));
```

Comment:
can you do Duration.ofMillis(10000). It is what we've been using for the most part

Reply:
Updated